app/controllers/report_controller.rb

```diff
@@ -699,20 +689,14 @@ def replace_right_cell(options = {})  # :replace_trees key can be an array of tr
 
     # With dynatree, simply replace the tree partials to reload the trees
     replace_trees.each do |t|
-      case t
-      when :reports
-        presenter[:replace_partials][:reports_tree_div] = r[:partial => "reports_tree"]
-      when :schedules
-        presenter[:replace_partials][:schedules_tree_div] = r[:partial => "schedules_tree"]
-      when :savedreports
-        presenter[:replace_partials][:savedreports_tree_div] = r[:partial => "savedreports_tree"]
-      when :db
-        presenter[:replace_partials][:db_tree_div] = r[:partial => "db_tree"]
-      when :widgets
-        presenter[:replace_partials][:widgets_tree_div] = r[:partial => "widgets_tree"]
```

Comment:
No problem :-) I was actually looking here to see what I may have forgotten in #3530 :-) 

Oh, but btw, isn't `@sb` supposed to be replaced by.. Uh, tree_state? But maybe you're handling it differently..